ios/NewExpensify/Info.plist

```diff
@@ -26,6 +26,7 @@
 			<key>CFBundleURLSchemes</key>
 			<array>
 				<string>expensify-cash</string>
```

Comment:
Do we still need the old URLScheme? Is there anywhere that still uses this?

Reply:
I'm not aware of any code that still uses this, but I know we have some code that links back-and-forth between OldDot and NewDot and I don't want to break anything in this PR by removing it.

Reply:
We can create a FP follow-up to remove this and any usages across all our repos (probably just E/App and Web-Expensify).

Reply:
https://github.com/Expensify/Expensify/issues/176081

Reply:
https://github.com/Expensify/Expensify/issues/176081